mmaction/apis/test.py

```diff
@@ -181,3 +180,4 @@ def collect_results_gpu(result_part, size):
         # the dataloader may pad some samples
         ordered_results = ordered_results[:size]
         return ordered_results
+    return None
```

Comment:
double check that None is a valid return value

Reply:
`None` is a valid return value by checking